src/main/java/hu/oe/nik/szfmv/automatedcar/model/WorldObject.java

```diff
@@ -133,6 +134,11 @@ public BufferedImage getImage() {
         return this.image;
     }
 
+    // will be added later by team 2 but needed for
```

Comment:
Teljesen jó így, de erről esetleg célszerű őket tájékoztatni. Ha beszéltetek erről, akkor tárgytalan.

Reply:
Az ő munkaközijükből néztük meg, hogy hogy kapom majd az adatot. És ki is próbáltuk az ő branchuk fájljaival, csak az nyilván nem mergeltem fel a sajátunkkal együtt, mert tanultam a multkori zavarosságból.